apps/nextjs-website/react-components/types/FramedVideo/FramedVideo.types.ts

```diff
@@ -0,0 +1,39 @@
+import { SectionProps } from '../common/Common.types';
+
+export interface FramedVideoProps extends SectionProps {
+  theme: 'light' | 'dark';
+  videoURL?: string | null;
+  video?:
+    | {
+        srcURL: string | null;
+        autoplay: boolean;
+        loop: boolean;
+        showControls: boolean;
+        fallback: string;
+        playButtonLabel: string;
+        pausedPlayButtonLabel: string;
+      }
+    | undefined;
+  text?: {
+    title: string;
+    body: string;
+    link: {
+      href: string;
+      label: string;
+    };
+    textPosition: 'left' | 'right';
+  } | null;
```

Comment:
```suggestion
  text?: {
    title: string;
    body: string;
    link: {
      href: string;
      label: string;
    };
    textPosition: 'left' | 'right';
  };
```
Field is already optional.
Never accept null as a value outside of sectionID.